src/pages/SignUp.jsx

```diff
@@ -17,13 +15,11 @@ export default function SignUp() {
   // follow this function for a better practice with interactive button
 
   const { run: signUp, loading: signUpLoading } = useRequest(
-    // fixme: temp hard code company
+    // No harding for company required as it will be entered by user when registering for first time
 
     async (data) =>
       auth.signUp({
         ...data,
-        // fixme: temp hard code company
-        company: doc(database, "Companies", "KJLOQ9jWh9zsc3JfBugR"),
       }),
     {
       manual: true,
```

Comment:
for this, help add a success toast on success.
onError, instead of console, add a fail toast

Reply:
Added toast message for success & failure